### PR TITLE
Fix json dump bug in NL logging

### PR DESCRIPTION
**Pull request by shifucun**
The new MultiVarCandidates can not be dumped by json, using pickle instead.

Also fix some other small issues.

### CODE REVIEW

**Review by chejennifer — APPROVED** [review]
thanks for the fixes!

File: server/services/bigtable.py [hunk 1]
```diff
@@ -87,7 +88,7 @@ async def write_row(session_info, data):
   row.set_cell(_COLUMN_FAMILY, _COL_PROJECT.encode(), project_id)
   row.set_cell(_COLUMN_FAMILY, _COL_SESSION.encode(), json.dumps(session_info))
   row.set_cell(_COLUMN_FAMILY, _COL_VERSION.encode(), json.dumps(version))
-  row.set_cell(_COLUMN_FAMILY, _COL_DATA.encode(), json.dumps(data))
+  row.set_cell(_COLUMN_FAMILY, _COL_DATA.encode(), pickle.dumps(data))
```
Comment (pradh) [inline server/services/bigtable.py:91]: Maybe revert just this change, and I'll make a PR as followup?

**Review by pradh — APPROVED** [review]

File: server/services/bigtable.py (hunk 1 above)
Comment (shifucun) [reply server/services/bigtable.py:91]: I will hold off this PR for other purpose then, since this was aimed to fix this bug.

Comment (pradh) [reply server/services/bigtable.py:91]: No, I don't think its from the recent change that added MultiSV.  Generally data_dict is not safe to convert to json.

So submit this change unbreak autopush for now, and I can look into why this is and switch to json?

**Review by pradh — APPROVED** [review]

File: server/services/bigtable.py (hunk 1 above)
Comment (shifucun) [reply server/services/bigtable.py:91]: sg!

**Pull request merged**